[2022-07-04 21:30:16,028] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-07-04T20:30:00+00:00 [queued]>
[2022-07-04 21:30:16,119] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-07-04T20:30:00+00:00 [queued]>
[2022-07-04 21:30:16,119] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 21:30:16,126] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-04 21:30:16,126] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 21:30:16,236] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): convert_xls_in_xlsx> on 2022-07-04 20:30:00+00:00
[2022-07-04 21:30:16,246] {standard_task_runner.py:52} INFO - Started process 190 to run task
[2022-07-04 21:30:16,256] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'convert_xls_in_xlsx', 'scheduled__2022-07-04T20:30:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpasmefwz7', '--error-file', '/tmp/tmplo3_3d2a']
[2022-07-04 21:30:16,260] {standard_task_runner.py:80} INFO - Job 33: Subtask convert_xls_in_xlsx
[2022-07-04 21:30:16,420] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-07-04T20:30:00+00:00 [running]> on host d63650f117fd
[2022-07-04 21:30:16,675] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=convert_xls_in_xlsx
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T20:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-04T20:30:00+00:00
